playground/frontend/lib/pages/playground/components/playground_feedback.dart

```diff
@@ -37,14 +38,20 @@ class PlaygroundFeedback extends StatelessWidget {
         ),
         IconButton(
           padding: EdgeInsets.zero,
-          // ignore: avoid_print
-          onPressed: () => print('Feedback Up'),
+          onPressed: () {
+            AnalyticsService.get(context).trackClickEnjoyPlayground(true);
+            // ignore: avoid_print
+            print('Feedback Up');
+          },
           icon: SvgPicture.asset(kThumbUpIconAsset),
         ),
         IconButton(
           padding: EdgeInsets.zero,
-          // ignore: avoid_print
-          onPressed: () => print('Feedback down'),
+          onPressed: () {
+            AnalyticsService.get(context).trackClickEnjoyPlayground(false);
+            // ignore: avoid_print
+            print('Feedback down');
```

Comment:
Same comment as above.  Could we remove this print statement?  Does the AnalyticsService throw any errors that need to be caught and logged?

Reply:
same here